i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments-administration/wal/remote-wal/manage-kafka.md

```diff
@@ -18,34 +18,42 @@ GreptimeDB 集群在启用 [Remote WAL](/user-guide/deployments-administration/w
 将以下内容保存为配置文件 `kafka.yaml`：
 
 ```yaml
-global:
-  security:
-    allowInsecureImages: true
-
-image:
-  registry: docker.io
-  repository: greptime/kafka
-  tag: 3.9.0-debian-12-r12
```

Comment:
Use greptime image.
https://hub.docker.com/repository/docker/greptime/kafka/general

Reply:
what's special about that kafka? `allowInsecureImages: true` often makes user suspicious